docker/pegasus-build-env/centos6/Dockerfile

```diff
@@ -20,16 +20,16 @@ FROM centos:6.10
 LABEL maintainer=jiashuo
 
 # defaut repo url hasn't been supported, use thirdparty repo url
-RUN curl https://www.getpagespeed.com/files/centos6-eol.repo --output /etc/yum.repos.d/CentOS-Base.repo
-RUN curl https://www.getpagespeed.com/files/centos6-epel-eol.repo --output /etc/yum.repos.d/epel.repo
-RUN curl https://www.getpagespeed.com/files/centos6-scl-eol.repo --output /etc/yum.repos.d/CentOS-SCLo-scl.repo
-RUN curl https://www.getpagespeed.com/files/centos6-scl-rh-eol.repo --output /etc/yum.repos.d/CentOS-SCLo-scl-rh.repo
+RUN wget --progress=dot:giga https://www.getpagespeed.com/files/centos6-eol.repo -O /etc/yum.repos.d/CentOS-Base.repo
```

Comment:
it seems not have `wget` tools for the new centos env

Reply:
1. Docker lint will warn both curl and wget used together. So avoid this should use only one.
2. curl get the file thrift-0.11.0.tgz, I failed. So I fix curl before to wget.
3. If wget is not exist. Keey curl and ignore the warnings in docker lint?

Reply:
-:81 DL4001 warning: Either use Wget or Curl but not both

Reply:
DONE

Reply:
> 2\. curl get the file thrift-0.11.0.tgz, I failed. So I fix curl before to wget.

thrift can be fetched by `wget`, but the centos6-epel-eol.repo...., should use `curl`

Or you can test, whether `wget` exist or not for the new centos6 env. I remember it don't have